app/services/metrics_publisher.rb

```diff
@@ -19,13 +19,14 @@ def profiles_report
 
   def completion_report
     report = { 'mean' => Person.overall_completion }
-    Person.bucketed_completion.each do |range, total|
-      report[range_to_string(range)] = total
+    report['completions_by_range'] = Person.bucketed_completion.map do |range, total|
+      { 'range' => range_inclusion_as_string(range),
+        'total' => total }
     end
     report
   end
 
-  def range_to_string(range)
-    '[%d,%d%s' % [range.begin, range.end, range.exclude_end? ? ')' : ']']
+  def range_inclusion_as_string(range)
+    range.end == 100 ? "#{range.begin} ≤ n ≤ 100" : "#{range.begin} ≤ n < #{range.end}"
```

Comment:
You could change `range.end == 100` to `range.exclude_end?` and swap the branches. Then it doesn't care about any external information.